requirements/default.txt

```diff
@@ -2,7 +2,7 @@ numpy>=1.16.5
 scipy>=1.2.3
 matplotlib>=3.0.3
 networkx>=2.2
-pillow>=5.4.0,!=7.1.0,!=7.1.1
+pillow>=8.1.2
```

Comment:
THen, I guess you want to relax the min version

Reply:
> see above for the min req
>  THen, I guess you want to relax the min version

Sorry @sciunto, I didn't get it :slightly_smiling_face:

Reply:
Don't you want to set >=6.1.0 or so? otherwise, the warning doesn't make sense anymore.

Reply:
I expected it to be >=8.1.2, but I modified the warning with the latest I'd like it to be >=8.1.2, but I updated the CVE numbers on the warning to reflect the latest issues. Isn't it the way to do here?
What would you prefer? To set up >=6.1.0?

Reply:
I expect it to be >=8.1.2, but I updated the CVE numbers on the warning to reflect the latest issues. Isn't it the way to do here?
What would you prefer? To set up >=6.1.0?

Reply:
If you choose > 8.1.2, no need to have the warning, because the users will be forced to upgrade. If we do not want to force the upgrade, then warning and >6.1.0. THat's my understanding.

Reply:
@sciunto got it. I thought the warning would check if the machine has that version, and would ask to update based on that.
Warning off, then? :slightly_smiling_face: